### DISCUSSION TITLE
random / shuffle

### DISCUSSION THREAD

**Discussion by awesome** [tailwindlabs/heroicons#534] [Ideas]
![download](https://user-images.githubusercontent.com/19291/139105870-57e2b225-1821-40cd-a4d9-cd67eed3b170.png)

**Comment by ndri** [comment]
I added `arrows-crossing` to [Sidekickicons](https://github.com/ndri/sidekickicons/), which are meant to be used as a companion to Heroicons. Browse them here: https://sidekickicons.com/?icon=arrows-crossing
